packages/typo3-guides-extension/src/Command/RunDecorator.php

```diff
@@ -0,0 +1,150 @@
+<?php
+
+namespace T3Docs\GuidesExtension\Command;
+
+use phpDocumentor\Guides\Cli\Command\Run;
+use Symfony\Component\Console\Application;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\ArrayInput;
+use Symfony\Component\Console\Input\InputDefinition;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+
+final class RunDecorator extends Command
+{
+    private const DEFAULT_OUTPUT_DIRECTORY = 'Documentation-GENERATED-temp';
+
+    private const INDEX_FILE_NAMES = [
```

Comment:
What about using one array like

    [
        'Index.rst',
        'index.rst',
        'Index.md',
        'index.md',
    ]
    
This way it is clearer in which order the index files are searched for. And we can omit one foreach loop in `guessInput()`.